packages/Interface/test/tstFortran_API.f90

```diff
@@ -700,11 +700,11 @@ program main
   use datatransferkit
   use x
   use utc
-#ifdef HAVE_MPI
   use mpi
-#endif
   implicit none
 
+  integer :: ierr
```

Comment:
Note that `HAVE_MPI` was never defined and that the code would not compiler because `ierr` was not declared anywhere.